lexnlp/extract/pt/__init__.py

```
@@ -0,0 +1,61 @@
+from lexnlp.extract.pt.copyrights import get_copyright_annotations, get_copyrights
+from lexnlp.extract.pt.courts import get_court_annotations, get_courts
+from lexnlp.extract.pt.dates import get_date_annotations, get_dates
+from lexnlp.extract.pt.definitions import get_definition_annotations, get_definitions
+from lexnlp.extract.pt.identifiers import (
+    get_cnpj_annotations,
+    get_cpf_annotations,
+    get_identifier_annotations,
+    get_oab_annotations,
+)
+from lexnlp.extract.pt.language_tokens import PtLanguageTokens
+from lexnlp.extract.pt.regulations import get_regulation_annotations, get_regulations
```

Pylint (4.0.5)

[error] Cannot import 'lexnlp.extract.pt.regulations' due to 'unexpected indent (lexnlp.extract.pt.regulations, line 173)'

(E0001)